src/WebApi/Controllers/UsersController.cs

```diff
@@ -210,6 +211,35 @@ public Task<ActionResult<Result<CharacterCharacteristicsViewModel>>> GetCharacte
         }));
     }
 
+    /// <summary>
+    /// Get character competitive rating.
+    /// </summary>
+    /// <param name="id">Character id.</param>
+    /// <returns>The character rating.</returns>
+    /// <response code="200">Ok.</response>
+    [HttpGet("self/characters/{id}/competitive-rating")]
```

Comment:
```suggestion
    [HttpGet("self/characters/{id}/rating")]
```

Since you are returning the full rating object.